src/main/java/com/google/api/gax/grpc/ApiCallable.java

```diff
@@ -236,6 +235,8 @@ public ApiCallSettings getSettings() {
    *
    * @param context {@link com.google.api.gax.grpc.CallContext} to make the call with
    * @return the call result
+   * @throws ApiException If there is any bad status in the response.
```

Comment:
* Use a separate full @throws line for each exception
* Always use the wording "if ..." after the exception class name (note: lower-case 'if')